server/notifications/tests/services_tests.py

```diff
@@ -0,0 +1,63 @@
+from notifications.tests import shared_testcase
+from notifications import services
+from notifications import models
+
+
+class ServicesTests(shared_testcase.SharedTestCase):
+
+    def setUp(self):
+        super(ServicesTests, self).setUp()
+        self.html_content = '<p>This is an html message</p>'
+
+    def test_create_multialternatives(self):
+        notification = models.EmailNotification(message=self.html_content,
+                                                html_email=True,
+                                                sending_user=self.superuser1,
+                                                subject='Subject')
+        notification.save()
+        notification.recipients.set([self.superuser2])
```

Comment:
Consider making this `self.notification` since it's also used in another test. Also, is there a reason why you made a notification from scratch here and used `self.create_emailnotification(...)` in other tests?

Reply:
I'll modify `create_emailnotification` to include `html_content` as a parameter, and re-use it.

Reply:
I'll modify `create_emailnotification` to include `html_content` as a parameter, and re-use it.

Reply:
Fixed.